setup.cfg

```diff
@@ -15,4 +15,36 @@ replace = __version__ = '{new_version}'
 universal = 1
 
 [flake8]
-exclude = docs
+ignore = E226,E302,E41
+max-line-length = 160
```

Comment:
Woooooooooooooooooo! So generous! And here I was trying to hold myself back to ~100!